includes/classes/SiteAutomation/Request.php

```diff
@@ -79,9 +79,13 @@ public function get( $page, $widget ) {
 		$this->api_url = $this->set_api_url( $page, $widget );
 
 		$this->status = $this->get_status();
-		$site_automation_data = get_option( "sophi_site_automation_data_{$page}_{$widget}" );
+		$site_automation_data = false;
```

Comment:
Currently, the return type is set to `array|null`, it seems it should be `array|bool`. Also, would it make sense to also have a third parameter to this function to skip cache or not instead of relying solely on a constant that will affect the behavior every time?